framework/include/constraints/AutomaticMortarGeneration.h

```diff
@@ -326,6 +326,14 @@ class AutomaticMortarGeneration : public ConsoleStreamInterface
    */
   const std::set<SubdomainID> & primaryIPSubIDs() const { return _primary_ip_sub_ids; }
 
+  /**
+   * @return Map from node id to secondary lower-d element pointer
+   */
+  const std::unordered_map<dof_id_type, std::vector<const Elem *>> nodesToSecondaryElem() const
```

Comment:
```suggestion
  const std::unordered_map<dof_id_type, std::vector<const Elem *>> & nodesToSecondaryElem() const
```
Don't want to make a copy here